lib/oauth.rb

```diff
@@ -1,7 +1,7 @@
 module Oauth
   SCOPES = %w[read_prefs write_prefs write_diary write_api read_gpx write_gpx write_notes].freeze
   PRIVILEGED_SCOPES = %w[read_email skip_authorization].freeze
-  OAUTH2_SCOPES = %w[openid].freeze
+  OAUTH2_SCOPES = %w[write_blocks openid].freeze
```

Comment:
I'm wondering if it would make sense to only allow moderators and admins to register a new app with "Create and revoke user blocks" permissions?

Reply:
Possibly. But on the other hand, moderator permissions come and go, and for example I could develop and register an app for use by moderators, despite not being a moderator (or admin) myself.

Reply:
It wouldn't. For example, I need this feature for [osm-revert-scripts](https://github.com/woodpeck/osm-revert-scripts). You can run these scripts on OpenHistoricalMap too. I've registered an app there, but I don't have mod/admin permissions.

Reply:
I agree it's more convenient for a developer. On the other hand, mods need to be extra careful when authorizing apps, since there will be no control on who can request this permission for their app.

Reply:
You can do redactions with just `write_api`, the permission any editor requires.

Reply:
Added a star next to `write_blocks` scope to make it more noticeable:
![image](https://github.com/openstreetmap/openstreetmap-website/assets/4158490/6e17434e-6115-4f6d-9984-82c5d99256de)

Reply:
addressed in #4387